supplementary_style_guide/master.adoc

```diff
@@ -40,6 +40,9 @@ include::style_guidelines/specific-doc-types.adoc[leveloffset=+2]
 // Links
 include::style_guidelines/links.adoc[leveloffset=+2]
 
+// Statements about the future
```

Comment:
Since this really is only a single thing and not necessarily a category that will have additional guidelines added to it, I think we probably don't need to make its own category. I'd recommend just adding it to the "General formatting" section's file, kind of like how the conscious language one is there.

Reply:
done